### DISCUSSION TITLE
Specify custom simulator domain

### DISCUSSION THREAD

**Discussion by bstiffler582** [thefrontside/simulacrum#250] [Q&A]
When trying to containerize the auth0 simulator and run it separately from my app, I need to be able to reference the service with a domain other than "localhost". It seems as though the meta data returned from e.g. "/.well-known/openid-configuration" is hard coded as "localhost". Does the tool provide a means to change this value? I need to return something like:
{"issuer":"https://host.docker.internal:4400/"... } or "https://auth0sim.local:4400, etc.
otherwise my app container will be referencing itself.
I believe this is the last piece I need, unless I am (quite likely) missing a bigger picture here...

**Comment by taras** [comment]
@bstiffler582 welcome. This is a legitimate use case. I created an issue to track this https://github.com/thefrontside/simulacrum/issues/251

Would you be open to contributing a fix?

**Reply by bstiffler582** [reply]
@taras I can pull the source and take a look when I have time, but this is not my native tech stack so I would probably make a mess of it 🙃.
